packages/ingest/s3.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const aws = require('@cumulus/common/aws');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const urljoin = require('url-join');
+
+module.exports.s3Mixin = (superclass) => class extends superclass {
+
+  /**
+   * Downloads a given url and upload to a given S3 location
```

Comment:
This makes it sounds like it's being fetched over HTTP.  "Copies an object from one S3 location to another" would be closer to what's happening.